src/redux/helpers.ts

```diff
@@ -0,0 +1,9 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from "react-redux";
+import { AppDispatch, RootState } from "./store";
+
+export const useAppDispatch = () => useDispatch<AppDispatch>()
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector
+
+export const selectAccessToken = (state: RootState) => state.auth.loginData.access.accessToken;
```

Comment:
- [ ] create selectors for the state.auth.loginData.access.accessToken and similar things